app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -262,5 +266,23 @@ def run_generic_operation(operation, ems_ref, parameters = {})
         end
       end
     end
+
+    def run_specific_operation(operation_name, ems_ref, parameters = {})
```

Comment:
Yes, I can merge in one method with optional parameters, I didn't do it because I think it will be confusing have operation and operation_name parameters on the same method.